app/tag-management/new-tag/new-tag.component.html

```diff
@@ -0,0 +1,42 @@
+  <div class="top-content">
+    <div class="valign">
+      <md-card>
```

Comment:
In all other components, we have an extra md-card for the title and using `h1` as headline. Please do it like that for your components as well to be consistent:
```
<md-card class="default-card">
   <h1>Title</h1>
</md-card>
```